Loop/Extensions/DeviceDataManager+BolusEntryViewModelDelegate.swift

```diff
@@ -20,10 +20,14 @@ extension DeviceDataManager: BolusEntryViewModelDelegate {
         loopManager.addGlucose(samples, completion: completion)
     }
     
-    func addCarbEntry(_ carbEntry: NewCarbEntry, replacing replacingEntry: StoredCarbEntry?, completion: @escaping (Result<Void>) -> Void) {
+    func addCarbEntry(_ carbEntry: NewCarbEntry, replacing replacingEntry: StoredCarbEntry?, completion: @escaping (Result<StoredCarbEntry>) -> Void) {
         loopManager.addCarbEntry(carbEntry, replacing: replacingEntry, completion: completion)
     }
 
+    func storeBolusDosingDecision(_ bolusDosingDecision: BolusDosingDecision, withDate date: Date) {
+        loopManager.storeBolusDosingDecision(bolusDosingDecision, withDate: date)
+    }
+
```

Comment:
Should this also have a `completion` handler like the rest?

Reply:
This routine is synchronous, so no completion handler is required.